### ISSUE TITLE
BBC: Blead breaks Webservice-InterMine 1.0405

### ISSUE THREAD

**Issue by andk** [Perl/perl5#20447]
**Description**

Starting with v5.37.4-130-g1c33371ae0 tests are failing for INTERMINE/Webservice-InterMine-1.0405.tar.gz

Sample fail report:
http://www.cpantesters.org/cpan/report/1792cd7e-550c-11ed-8d72-38236cca80ec

Git bisect for this fail lead me to
``` text
     1c33371ae09893c7dcf660df882cc1a5dbc37266 is the first bad commit                         
     commit 1c33371ae09893c7dcf660df882cc1a5dbc37266
     Author: Karl Williamson <khw@cpan.org>
     Date:   Tue Sep 27 15:51:16 2022 -0600
     
         sv.c: Clone interpreter with locale set to C
```

**Steps to Reproduce**

cpan -i  INTERMINE/Webservice-InterMine-1.0405.tar.gz

**Expected behavior**

Should build, test and install the pkg

**Perl configuration**

```
# perl -V output goes here
Summary of my perl5 (revision 5 version 37 subversion 5) configuration:
  Commit id: 064941484781d9099db0bbe7d6ea22a826df1e15
  Platform:
    osname=linux
    osvers=5.15.0-50-generic
    archname=x86_64-linux-thread-multi-ld
    uname='linux k93jammy 5.15.0-50-generic #56-ubuntu smp tue sep 20 13:23:26 utc 2022 x86_64 x86_64 x86_64 gnulinux '
    config_args='-Dprefix=/home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.37.4-160-g0649414847/322f -Dmyhostname=k93jammy -Dinstallusrbinperl=n -Uversiononly -Dusedevel -des -Ui_db -Dlibswanted=cl pthread socket inet nsl gdbm dbm malloc dl ld sun m crypt sec util c cposix posix ucb BSD gdbm_compat -Duseithreads -Duselongdouble -DEBUGGING=-g'
    hint=recommended
    useposix=true
    d_sigaction=define
    useithreads=define
    usemultiplicity=define
    use64bitint=define
    use64bitall=define
    uselongdouble=define
    usemymalloc=n
    default_inc_excludes_dot=define
  Compiler:
    cc='cc'
    ccflags ='-D_REENTRANT -D_GNU_SOURCE -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64'
    optimize='-O2 -g'
    cppflags='-D_REENTRANT -D_GNU_SOURCE -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include'
    ccversion=''
    gccversion='11.2.0'
    gccosandvers=''
    intsize=4
    longsize=8
    ptrsize=8
    doublesize=8
    byteorder=12345678
    doublekind=3
    d_longlong=define
    longlongsize=8
    d_longdbl=define
    longdblsize=16
    longdblkind=3
    ivtype='long'
    ivsize=8
    nvtype='long double'
    nvsize=16
    Off_t='off_t'
    lseeksize=8
    alignbytes=16
    prototype=define
  Linker and Libraries:
    ld='cc'
    ldflags =' -fstack-protector-strong -L/usr/local/lib'
    libpth=/usr/local/lib /usr/lib/x86_64-linux-gnu /usr/lib /usr/lib64
    libs=-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc
    perllibs=-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc
    libc=/lib/x86_64-linux-gnu/libc.so.6
    so=so
    useshrplib=false
    libperl=libperl.a
    gnulibc_version='2.35'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs
    dlext=so
    d_dlsymun=undef
    ccdlflags='-Wl,-E'
    cccdlflags='-fPIC'
    lddlflags='-shared -O2 -g -L/usr/local/lib -fstack-protector-strong'


Characteristics of this binary (from libperl): 
  Compile-time options:
    HAS_TIMES
    MULTIPLICITY
    PERLIO_LAYERS
    PERL_COPY_ON_WRITE
    PERL_DONT_CREATE_GVSV
    PERL_HASH_FUNC_SIPHASH13
    PERL_HASH_USE_SBOX32
    PERL_MALLOC_WRAP
    PERL_OP_PARENT
    PERL_PRESERVE_IVUV
    PERL_USE_DEVEL
    PERL_USE_SAFE_PUTENV
    USE_64_BIT_ALL
    USE_64_BIT_INT
    USE_ITHREADS
    USE_LARGE_FILES
    USE_LOCALE
    USE_LOCALE_COLLATE
    USE_LOCALE_CTYPE
    USE_LOCALE_NUMERIC
    USE_LOCALE_TIME
    USE_LONG_DOUBLE
    USE_PERLIO
    USE_PERL_ATOF
    USE_REENTRANT_API
    USE_THREAD_SAFE_LOCALE
  Built under linux
  Compiled at Oct 21 2022 02:40:46
  %ENV:
    PERL="/tmp/basesmoker-reloperl-h8f7/bin/perl"
    PERL5LIB="/tmp/loop_over_bdir-1329634-ggBFlW/Alien-SNMP-MIBDEV-2.020000-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Alien-SNMP-MIBDEV-2.020000-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Sysync-0.35-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Sysync-0.35-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/MP3-PodcastFetch-1.05-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/MP3-PodcastFetch-1.05-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/CGI-Application-NetNewsIface-0.0204-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/CGI-Application-NetNewsIface-0.0204-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/triceps-2.1.0-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/triceps-2.1.0-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Business-Shipping-DataFiles-1.02-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Business-Shipping-DataFiles-1.02-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Debian-Packages-0.03-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Debian-Packages-0.03-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/RDF-NLP-SPARQLQuery-0.1-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/RDF-NLP-SPARQLQuery-0.1-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/downsexta-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/downsexta-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/AllKnowingDNS-1.7-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/AllKnowingDNS-1.7-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Net-OpenVPN-DDNS-0.1-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Net-OpenVPN-DDNS-0.1-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/App-oo_modulino_zsh_completion_helper-0.04-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/App-oo_modulino_zsh_completion_helper-0.04-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Device-Kiln-0.03-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Device-Kiln-0.03-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Linux-InitFS-0.2-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Linux-InitFS-0.2-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/ProjectBuilder-0.14.1-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/ProjectBuilder-0.14.1-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/EJBCA-CrlPublish-0.60-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/EJBCA-CrlPublish-0.60-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/makepp-1.19-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/makepp-1.19-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Devel-Required-0.16-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Devel-Required-0.16-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Lingua-BioYaTeA-0.11-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Lingua-BioYaTeA-0.11-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Lingua-YaTeA-0.626-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Lingua-YaTeA-0.626-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/shishi-0.0.1-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/shishi-0.0.1-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Dpkg-1.21.9-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Dpkg-1.21.9-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/dta-tokwrap-0.98-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/dta-tokwrap-0.98-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/DBIx-Migration-Directories-0.12-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/DBIx-Migration-Directories-0.12-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/Simulation-Automate-1.0.1-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/Simulation-Automate-1.0.1-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/CORBA-Python-2.66-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/CORBA-Python-2.66-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/App-rmcd-0.280-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/App-rmcd-0.280-0/blib/lib:/tmp/loop_over_bdir-1329634-ggBFlW/App-nrun-v1.1.2-0/blib/arch:/tmp/loop_over_bdir-1329634-ggBFlW/App-nrun-v1.1.2-0/blib/lib"
    PERL5OPT=""
    PERL5_CPANPLUS_IS_RUNNING="1329653"
    PERL5_CPAN_IS_RUNNING="1329653"
    PERL_CANARY_STABILITY_NOPROMPT="1"
    PERL_MM_USE_DEFAULT="1"
    PERL_USE_UNSAFE_INC="1"
  @INC:
    /tmp/loop_over_bdir-1329634-ggBFlW/Alien-SNMP-MIBDEV-2.020000-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Alien-SNMP-MIBDEV-2.020000-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Sysync-0.35-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Sysync-0.35-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/MP3-PodcastFetch-1.05-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/MP3-PodcastFetch-1.05-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/CGI-Application-NetNewsIface-0.0204-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/CGI-Application-NetNewsIface-0.0204-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/triceps-2.1.0-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/triceps-2.1.0-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Business-Shipping-DataFiles-1.02-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Business-Shipping-DataFiles-1.02-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Debian-Packages-0.03-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Debian-Packages-0.03-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/RDF-NLP-SPARQLQuery-0.1-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/RDF-NLP-SPARQLQuery-0.1-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/downsexta-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/downsexta-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/AllKnowingDNS-1.7-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/AllKnowingDNS-1.7-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Net-OpenVPN-DDNS-0.1-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Net-OpenVPN-DDNS-0.1-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/App-oo_modulino_zsh_completion_helper-0.04-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/App-oo_modulino_zsh_completion_helper-0.04-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Device-Kiln-0.03-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Device-Kiln-0.03-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Linux-InitFS-0.2-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Linux-InitFS-0.2-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/ProjectBuilder-0.14.1-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/ProjectBuilder-0.14.1-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/EJBCA-CrlPublish-0.60-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/EJBCA-CrlPublish-0.60-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/makepp-1.19-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/makepp-1.19-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Devel-Required-0.16-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Devel-Required-0.16-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Lingua-BioYaTeA-0.11-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Lingua-BioYaTeA-0.11-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Lingua-YaTeA-0.626-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Lingua-YaTeA-0.626-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/shishi-0.0.1-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/shishi-0.0.1-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Dpkg-1.21.9-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Dpkg-1.21.9-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/dta-tokwrap-0.98-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/dta-tokwrap-0.98-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/DBIx-Migration-Directories-0.12-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/DBIx-Migration-Directories-0.12-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/Simulation-Automate-1.0.1-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/Simulation-Automate-1.0.1-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/CORBA-Python-2.66-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/CORBA-Python-2.66-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/App-rmcd-0.280-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/App-rmcd-0.280-0/blib/lib
    /tmp/loop_over_bdir-1329634-ggBFlW/App-nrun-v1.1.2-0/blib/arch
    /tmp/loop_over_bdir-1329634-ggBFlW/App-nrun-v1.1.2-0/blib/lib
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.37.4-160-g0649414847/322f/lib/site_perl/5.37.5/x86_64-linux-thread-multi-ld
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.37.4-160-g0649414847/322f/lib/site_perl/5.37.5
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.37.4-160-g0649414847/322f/lib/5.37.5/x86_64-linux-thread-multi-ld
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.37.4-160-g0649414847/322f/lib/5.37.5
    .
```

**Comment by jkeenan** [comment]
The test file in Webservice-InterMine which is failing does so on the very first unit test, which is a `use_ok` call.  But there's a lot of set-up code before we get to this test:
```
[perlmonger: Webservice-InterMine-1.0405-0] $ head -n 87 t/07_intermine/01_integration.t.orig
use strict;
use warnings;
use Carp qw(confess cluck);
BEGIN {
    eval "use YAML::Syck";
    $ENV{TEST_YAML} = ($@) ? 0 : 1;
}
$SIG{__DIE__} = \&Carp::confess;
$SIG{__WARN__} = \&Carp::cluck;

use lib 't/tests'; # for the test role FooBar
use Test::More tests => 44;
use Test::MockObject::Extends;
use Test::Exception;
use HTTP::Response;
use IO::File;
use InterMine::Model::TestModel;
use XML::Compare;

sub slurp {
    my $file = shift;
    return join('', IO::File->new($file, 'r')->getlines);
}

my $module = 'Webservice::InterMine';

my $results_file = 't/data/mock_content_results';

my $model     = InterMine::Model::TestModel->instance->to_xml;
my $results   = slurp($results_file);
my $templates = slurp('t/data/default-template-queries.xml');

my $fake_lwp = Test::MockObject::Extends->new('LWP::UserAgent');
$fake_lwp->fake_module(
    'LWP::UserAgent',
    new => sub {
	return $fake_lwp;
    },
);
$fake_lwp->mock(
    get => sub {
	my $self = shift;
	my $uri  = shift;
	my ($code, $msg, $head, $content) = (200, 'OK', [foo => 'bar', 'Content-Location' => $uri]);
	if ($uri =~ m!templates!) {
	    $content = $templates;
	}
	elsif ($uri =~ m!/model!) {
	    $content = $model;
	}
	elsif ($uri =~ m!version!) {
	    $content = 4;
	}
	elsif ($uri =~ m!/results\?!) {
	    $content = $results;
	}
	my $r = HTTP::Response->new( $code, $msg, $head, $content );
	return $r;
    },
);
my $fake_IOSock = Test::MockObject::Extends->new();
$fake_IOSock->set_isa('Net::HTTP');
$fake_IOSock->fake_module(
    'Net::HTTP',
    new => sub {
        delete $fake_IOSock->{io};
        $fake_IOSock->{io} = IO::File->new($results_file, 'r');
        return $fake_IOSock;
    },
);
$fake_IOSock->mock(
    getline => sub {
	my $self = shift;
	return $self->{io}->getline;
    },
);
$fake_IOSock->mock(write_request => sub { return 1; });
$fake_IOSock->mock(
    close => sub {
	my $self = shift;
	$self->{io}->close;
    },
);

my $url = 'fake.url/path';
my @view = ('Employee.name', 'Employee.age', 'Employee.fullTime');
use_ok($module, ($url));
```

If I comment out all unit tests after the first, this is what the failure looks like:
```
$ bleadprove -vb t/07_intermine/01_integration.t
t/07_intermine/01_integration.t .. 
not ok 1 - use Webservice::InterMine;

#   Failed test 'use Webservice::InterMine;'
#   at t/07_intermine/01_integration.t line 87.
#     Tried to use 'Webservice::InterMine'.
#     Error:  Can't call method "host" on an undefined value at /usr/home/jkeenan/.cpan/build/Webservice-InterMine-1.0405-0/blib/lib/Webservice/InterMine/Service.pm line 833.
#  at /usr/home/jkeenan/.cpan/build/Webservice-InterMine-1.0405-0/blib/lib/Webservice/InterMine/Service.pm line 833.
# 	Webservice::InterMine::Service::fetch(Webservice::InterMine::Service=HASH(0x80481f4b0), "http://fake.url/path/service/version") called at /usr/home/jkeenan/.cpan/build/Webservice-InterMine-1.0405-0/blib/lib/Webservice/InterMine/Service.pm line 219
# 	Webservice::InterMine::Service::__ANON__(Webservice::InterMine::Service=HASH(0x80481f4b0)) called at constructor Webservice::InterMine::Service::new (defined at /usr/home/jkeenan/.cpan/build/Webservice-InterMine-1.0405-0/blib/lib/Webservice/InterMine/Service.pm line 913) line 141
# 	Webservice::InterMine::Service::new("Webservice::InterMine::Service", "fake.url/path") called at /usr/home/jkeenan/.cpan/build/Webservice-InterMine-1.0405-0/blib/lib/Webservice/InterMine.pm line 463
# 	Webservice::InterMine::get_service("Webservice::InterMine", "fake.url/path") called at /usr/home/jkeenan/.cpan/build/Webservice-InterMine-1.0405-0/blib/lib/Webservice/InterMine.pm line 207
# 	Webservice::InterMine::import("Webservice::InterMine", "fake.url/path") called at t/07_intermine/01_integration.t line 87
# 	main::BEGIN() called at t/07_intermine/01_integration.t line 87
# 	eval {...} called at t/07_intermine/01_integration.t line 87
# 	eval 'package main;
# BEGIN { ${^WARNING_BITS} = $args[-1] if defined $args[-1] }
# #line 87 t/07_intermine/01_integration.t
# use Webservice::InterMine @{$args[0]};
# 1;
# ' called at /home/jkeenan/testing/blead/lib/perl5/5.37.6/Test/More.pm line 1034
# 	Test::More::_eval("package main;\x{a}BEGIN { \${^WARNING_BITS} = \$args[-1] if defined"..., ARRAY(0x8019aa7f8), "UUUUUUUUUUUUUUUUUUU") called at /home/jkeenan/testing/blead/lib/perl5/5.37.6/Test/More.pm line 1009
# 	Test::More::use_ok("Webservice::InterMine", "fake.url/path") called at t/07_intermine/01_integration.t line 87
# BEGIN failed--compilation aborted at t/07_intermine/01_integration.t line 87.
#  at t/07_intermine/01_integration.t line 87.
# 	eval 'package main;
# BEGIN { ${^WARNING_BITS} = $args[-1] if defined $args[-1] }
# #line 87 t/07_intermine/01_integration.t
# use Webservice::InterMine @{$args[0]};
# 1;
# ' called at /home/jkeenan/testing/blead/lib/perl5/5.37.6/Test/More.pm line 1034
# 	Test::More::_eval("package main;\x{a}BEGIN { \${^WARNING_BITS} = \$args[-1] if defined"..., ARRAY(0x8019aa7f8), "UUUUUUUUUUUUUUUUUUU") called at /home/jkeenan/testing/blead/lib/perl5/5.37.6/Test/More.pm line 1009
# 	Test::More::use_ok("Webservice::InterMine", "fake.url/path") called at t/07_intermine/01_integration.t line 87
1..1
# Looks like you failed 1 test of 1.
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 

Test Summary Report
-------------------
t/07_intermine/01_integration.t (Wstat: 256 (exited 1) Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
Files=1, Tests=1,  1 wallclock secs ( 0.04 usr  0.01 sys +  1.09 cusr  0.11 csys =  1.25 CPU)
Result: FAIL
```

**Comment by jkeenan** [comment]
> The test file in Webservice-InterMine which is failing does so on the very first unit test, which is a `use_ok` call. But there's a lot of set-up code before we get to this test:
> 

I have been able to reproduce the test failure on a threaded build on FreeBSD and on an unthreaded build on Linux.  So thread vs. non-thread does not appear to be the issue here.

**Comment by haarg** [comment]
This looks like a failure due to a change in HTTP::Message, not blead.

The test is expecting the `Content-Location` header to set the base URI, but this is no longer the case. This was changed in libwww-perl/HTTP-Message#185 and released in HTTP-Message 6.42.

**Comment by jkeenan** [comment]
> This looks like a failure due to a change in HTTP::Message, not blead.
> 
> The test is expecting the `Content-Location` header to set the base URI, but this is no longer the case. This was changed in [libwww-perl/HTTP-Message#185](https://github.com/libwww-perl/HTTP-Message/pull/185) and released in HTTP-Message 6.42.

That GH commit you cite actually points to HTTP::Response.  Nonetheless, when I reverted that change in my recently built blead perl and then called `test Webservice::InterMine` in the `cpan` shell, the test passed.  That confirms that this is not due to a change in blead.  Thanks for your investigation.  Closing this ticket.
